src/tests/ftest/util/job_manager_utils.py

```diff
@@ -126,6 +121,38 @@ def assign_environment_default(self, env_vars):
         """
         pass
 
+    def get_subprocess_state(self, message=None):
+        """Display the state of the subprocess.
+
+        Args:
+            message (str, optional): additional text to include in output.
+                Defaults to None.
+
+        Returns:
```

Comment:
If the patch needs any other changes, consider clarifying the comment.
if I understand the logic correctly
Will this return a state list of 0, 1, or 2 entries (and not more than 2)?
0: no job manager (orterun) found running and no remote processes found on _hosts?
1: job manager (orterun) found running, and no remote processes found on _hosts?
2(scenario a): job manager (orterun) found running, and some remote processes found on _hosts (but not specifically identifying the number of procs running and on which hosts)? 
2(scenario b): no job manager found running (state includes "?" for this) and somehow some remote procs found running on _hosts.

Reply:
The `super(JobManager, self).get_subprocess_state(message)` may return `None` or a list with 1 or more process states.  Then if this command was run as a sub process, it has an internal list of hosts defined, and there are processes found that match the command that this class is managing, that list will be extended by one more value (a previous value of `None` will be converted into a `["R"]` list).

Keep in mind that this is a common method for multiple `JobManager` based objects, but when used with an `Orterun` object that has been configured to manage the `daos_server` command, then the possible returns are:
- `None` - the local orterun process is dead (or something went wrong with the `ps` command) and there are no remote `daos_server` or `daos_io_server` processes running (or something went wrong with the pgrep command)
- `["R"]` - the local orterun process is dead and there is at least one remote `daos_server` or `daos_io_server` processes running OR the local orterun process is in the "R" state and there are no remote processes running.
- `[<state>]` - the local orterun process is running in some non-"R" state and there are no remote processes running
- `[<state1>, ..., "R"]` - the local orterun process is running in <state1> state and there are remote processes running - there may be additional states listed between these values if the `ps --forest` yields more than one pid, but I have not come across this in testing so far.